docs/docs/detecting/golang.md

```diff
@@ -70,7 +70,7 @@ There are 2 ways you can get your own golang signatures working with tracee.
         
         		switch e.EventName {
         		case "openat", "execve":
-        			arg, err := helpers.GetTraceeArgumentByName(e, "pathname")
+        			arg, err := helpers.GetInitializedTraceeArgumentByName(e, "pathname")
```

Comment:
This function name is too long.
Maybe rename to `GetInitializedEvtArg` and the existing `GetTraceeArgumentByName` function name to `GetEvtArg`